docs/DocGenerator/README.md

```diff
@@ -0,0 +1,205 @@
+# Wazuh QA DocGenerator
+Wazuh - Quality assurance automation self-contained documentation parsing tool
+
+## Rational
+Wazuh QA documentation is designed to be self-contained into the source code of each test.
+
+DocGenerator is the tool in charge of parsing the documentation block from each source code and generate data capable
+to be indexed and displayed.
+
+## Design
+### Input
+DocGenerator parses the information from a QA repository with test files containing a specific comment format:
+
+Each test file has self-contained documentation comment blocks in **YAML** format.
+
+These blocks can contain **Mandatory**, **Optional**, and **Ignored** fields.
```

Comment:
`may` instead of `can`?

Reply:
I consider **can** as more deterministic in this case.